composer.json

```diff
@@ -32,7 +32,7 @@
     "require": {
         "php": "^7.3 || ^8.0",
         "silverstripe/vendor-plugin": "^1",
-        "silverstripe/framework": "^4.10",
+        "silverstripe/framework": "^4.13",
```

Comment:
This is going to cause problems for people pinned to older versions of both fluent and framework. fluent 4 is very old at this point (most people are using 5 or 6). 

When there were risky updates (although strictly non-breaking changes) I took the conservative approach and just released a new major fluent version (hence fluent 5 and 6 are both silverstripe 4).

Yes to this change, however, on the 6 branch and above, since it's the current "active" feature branch, and users can realistically expect more active development on these branches.

I propose we leave fluent 4 as is. I also regret updating 4 to require 4.10. I think 5.x is still pinned at ^4 framework.

Reply:
> This is going to cause problems for people pinned to older versions of both fluent and framework. fluent 4 is very old at this point (most people are using 5 or 6).

The whole point of this is for people who are doing upgrades. For anyone _not_ upgrading to a more recent version of silverstripe/framework, they have no need of tractorcow/silverstripe-fluent 4.8 anyway. When they run `composer update tractorcow/silverstripe-fluent` this version won't even be presented to them as an option, so it won't cause any problems. I can't see any way in which this will cause problems for anyone, given composer has really good dependency resolution.